types/node/fs.d.ts

```diff
@@ -2258,6 +2276,24 @@ declare module 'fs' {
         position: ReadPosition | null,
         callback: (err: NodeJS.ErrnoException | null, bytesRead: number, buffer: TBuffer) => void
     ): void;
+    /**
+     * Similar to the above `fs.read` function, this version takes an optional `options` object.
+     * If not otherwise specified in an `options` object,
+     * `buffer` defaults to `Buffer.alloc(16384)`,
+     * `offset` defaults to `0`,
+     * `length` defaults to `buffer.byteLength`,  `- offset` after https://github.com/nodejs/node/pull/40349 is merged
```

Comment:
I think this comment could probably be updated to reflect that it's Node v17.6.0 that has change this, if desired (as the PR has been merged since this PR was submitted).